From: =?utf-8?q?Rog=C3=A9rio_Brito?= <rbrito@gmail.com>
Date: Sun, 22 Nov 2020 20:06:55 -0300
Subject: Makefile: Don't run flake8 when running offlinetest.

Unfortunately, due to the dynamic nature of python, we most certainly would
like to run flake8 when testing the package. Alas, flake8 sometimes
generates style-only warning or "errors" that break the build, in a similar
fashion to what -Werror does to GCC or other compilers.

Therefore, I'm disabling this.
---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -78,8 +78,8 @@ test:
 	$(PYTHON) -m pytest -Werror
 	$(MAKE) codetest
 
-offlinetest: codetest
-	$(PYTHON) -m pytest -Werror -m "not download"
+offlinetest:
+	$(PYTHON) -m pytest -m "not download"
 
 CODE_FOLDERS_CMD = find yt_dlp -type f -name '__init__.py' | sed 's,/__init__.py,,' | grep -v '/__' | sort
 CODE_FOLDERS != $(CODE_FOLDERS_CMD)
